lib/assessment/rubric_tester.py

```diff
@@ -303,13 +313,11 @@ def main():
                 logging.error(e)
 
         # download experiment files
-        if not os.path.exists(experiment_lesson_prefix) or options.download:
-            check_aws_access()
+        if not os.path.exists(current_experiment_dir) or options.download:
```

Comment:
ok, so it only clones if there is not already an experiment_dir for that branch? So it will not update if there are new commits on the remote branch, you will have to cd to the cloned repo in experiments and `git pull`, right?

Reply:
oh wait, you can add -d to re-clone